src/amr_criteria/refinement_package.hpp

```diff
@@ -37,14 +37,24 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin);
 template <typename T>
 TaskStatus Tag(T *rc);
 
-AmrTag CheckAllRefinement(MeshBlockData<Real> *rc);
+AmrTag CheckAllRefinement(MeshBlockData<Real> *rc,
+                          const AmrTag &level = AmrTag::derefine);
```

Comment:
What is the default level tag for?

Reply:
It was to avoid breaking things calling it the old way, but looking more closely I think that is only `MeshRefinement::CheckRefinementCondition()`, which can probably be removed now.